deploy/example_catalog/cr-app-tensorflow-notebook.json

```diff
@@ -2,36 +2,50 @@
     "apiVersion": "kubedirector.hpe.com/v1beta1",
     "kind": "KubeDirectorApp",
     "metadata": {
-        "name": "ml-jupyter-notebook",
-        "annotations": {
-            "token":"execute this command to get the authentication token 'kubectl exec <pod-name> jupyter notebook list' and use this token in Jupyter notebook."
-        }
+        "name" : "jupyter-notebook"
     },
 
-    "spec": {
+    "spec" : {
         "systemdRequired": true,
         "config": {
             "roleServices": [
                 {
                     "serviceIDs": [
+                        "ssh",
                         "jupyter-nb"
                     ],
                     "roleID": "controller"
+                },
+                {
+                    "serviceIDs": [
+                        "ssh"
+                    ],
+                    "roleID": "worker"
                 }
             ],
             "selectedRoles": [
-                "controller"
+                "controller",
+                "worker"
             ]
         },
         "label": {
-            "name": "TensorFlow + Jupyter",
-            "description": "TensorFlow GPU with Jupyter notebook"
+            "name": "Tensorflow distributed 1.9 on CPU with jupyter notebook & centos7x",
```

Comment:
looks like you made changes to old version, you should use this json as base and make your changes